<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_re2</class>
 <widget class="QDialog" name="Dialog_re2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>待归还名单：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>441</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>190</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>待归还盒子的联系方式：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>70</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>quit to back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
